Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6184.xaml

```diff
@@ -0,0 +1,98 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<controls:TestShell
+    xmlns:controls="clr-namespace:Xamarin.Forms.Controls"
+    xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:local="clr-namespace:Xamarin.Forms.Controls.Issues"
+    mc:Ignorable="d"
+             x:Class="Xamarin.Forms.Controls.Issues.Issue6184">
+    <ShellItem>
+        <ShellSection Title="Issue 1" IsEnabled="False">
+            <ShellContent>
+                <local:Issue1900/>
```

Comment:
Can you just use a page local to this test that gives instructions on how to make sure the test is passing?

Reply:
Yes, I'll work on this.